keyboards/vortex/m0110a/info.json

```diff
@@ -0,0 +1,493 @@
+{
+    "manufacturer": "Vortex",
+    "keyboard_name": "M0110A",
+    "maintainer": "Gsky",
+    "bootloader": "wb32-dfu",
+    "bootmagic": {
+        "matrix": [1, 6]
+    },
+    "diode_direction": "COL2ROW",
+    "dynamic_keymap": {
+        "layer_count": 5
+    },
+    "encoder": {
+        "enabled": false
+    },
+    "features": {
+        "audio": false,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "indicators": {
+        "num_lock": "C14",
+        "caps_lock": "C15"
+    },
+    "matrix_pins": {
+        "cols": ["A0", "A1", "A2", "A3", "A4", "A8", "A9", "A10"],
+        "rows": ["B0", "B1", "B2", "B3", "B4", "B5", "B6", "B7", "B8", "B9", "B10", "B11", "B12", "B13"]
+    },
+    "processor": "WB32FQ95",
+    "qmk": {
+        "tap_keycode_delay": 10
+    },
+    "url": "",
+    "usb": {
+        "device_version": "0.0.2",
+        "pid": "0xE39D",
+        "vid": "0x342D"
+    },
+    "eeprom": {
+        "driver": "wear_leveling"
+    },
+    "layouts": {
+        "LAYOUT_all": {
+                "layout": [
+                    { "label": "ESC", "matrix": [1, 6], "x": 0, "y": 0 },
+                    { "label": "1", "matrix": [1, 7], "x": 1, "y": 0 },
+                    { "label": "2", "matrix": [2, 7], "x": 2, "y": 0 },
+                    { "label": "3", "matrix": [3, 7], "x": 3, "y": 0 },
+                    { "label": "4", "matrix": [4, 7], "x": 4, "y": 0 },
+                    { "label": "5", "matrix": [4, 6], "x": 5, "y": 0 },
+                    { "label": "6", "matrix": [5, 6], "x": 6, "y": 0 },
+                    { "label": "7", "matrix": [5, 7], "x": 7, "y": 0 },
+                    { "label": "8", "matrix": [6, 7], "x": 8, "y": 0 },
+                    { "label": "9", "matrix": [7, 7], "x": 9, "y": 0 },
+                    { "label": "0", "matrix": [8, 7], "x": 10, "y": 0 },
+                    { "label": "-", "matrix": [8, 6], "x": 11, "y": 0 },
+                    { "label": "=", "matrix": [6, 6], "x": 12, "y": 0 },
+                    { "label": "\\", "matrix": [10, 2], "x": 13, "y": 0 },
+                    { "label": "delete", "matrix": [6, 5], "x": 14, "y": 0 },
+                    { "label": "Backspace", "matrix": [10, 1], "x": 15, "y": 0},
```

Comment:
top row in `LAYOUT_all` has one key too many, please resolve.
<img width="513" alt="LAYOUT_all" src="https://github.com/qmk/qmk_firmware/assets/75046609/a5253c29-5332-45e6-878c-b6c758f44991">